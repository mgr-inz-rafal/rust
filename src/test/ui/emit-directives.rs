// A very basic test for the emission of build directives in JSON output.

// compile-flags:--emit=metadata --error-format=json -Z emit-directives
// compile-pass

fn main() {}
