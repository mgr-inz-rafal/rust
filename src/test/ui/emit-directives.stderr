{"directive":"metadata file written: $TEST_BUILD_DIR/emit-directives/a"}
